### ISSUE TITLE
bug: Set version workflow fails

### ISSUE THREAD

**Issue by aninusmuffin** [ParadigmMC/mc-modpack-kit#42]
**Describe the bug**
When running the set version workflow it fails

**To Reproduce**
1. Go to Actions
2. Run the Set version | Manual Workflow
3. Fails

**Additional context**
```
Run git commit --allow-empty -m "chore: release v0.3.0" -m "Release-As: v0.3.0
  git commit --allow-empty -m "chore: release v0.3.0" -m "Release-As: v0.3.0
  shell: /usr/bin/bash -e {0}
/home/runner/work/_temp/bfbe7fc6-da1d-4d30-8d27-685ff7f79927.sh: line 1: unexpected EOF while looking for matching `"'
Error: Process completed with exit code 2.`
```

Oops

**Comment by sofiedotcafe** [comment]
Oh sorry
I notify you when I have done the rework/multibranch, it's bit broken rn

**Comment by sofiedotcafe** [comment]
Fixed with 00e966d
